src/i18n/en.json

```diff
@@ -964,49 +964,87 @@
-  "actionSidebar.tooltip.transferFunds.from": "Team the funds should be transferred from.",
-  "actionSidebar.tooltip.transferFunds.to": "Team the funds should be transferred to.",
-  "actionSidebar.tooltip.transferFunds.amount": "The amount of funds to be transferred.",
-  "actionSidebar.tooltip.mintTokens.amount": "The total amount of new tokens to be created.",
-  "actionSidebar.tooltip.createTeam.teamName": "The name for the team you want to create.",
-  "actionSidebar.tooltip.createTeam.teamPurpose": "Describe the purpose of the team.",
-  "actionSidebar.tooltip.createTeam.teamColour": "Select a color to represent the team for easier identification.",
-  "actionSidebar.tooltip.editTeam.selectTeam": "The name of the team you want to modify.",
-  "actionSidebar.tooltip.editTeam.teamName": "The new name for the team you are modifying.",
-  "actionSidebar.tooltip.manageReputation.team": "The team you would like reputation to be managed in.",
-  "actionSidebar.tooltip.manageReputation.modification": "Whether to Add or Remove the members reputation.",
-  "actionSidebar.tooltip.manageReputation.member": "The member who's reputation you want to manage.",
-  "actionSidebar.tooltip.manageReputation.currentAmount": "Their current amount of reputation.",
-  "actionSidebar.tooltip.manageReputation.newAmount": "The proposed new amount of reputation.",
-  "actionSidebar.tooltip.managePermissions.member": "Select the member whose permissions will be modified.",
-  "actionSidebar.tooltip.managePermissions.team": "Specify the team for the permission changes.",
-  "actionSidebar.tooltip.managePermissions.permissions": "Choose the permission for the selected member. View permissions list. INSTRUCTIONS: View permissions list - should open the modal.",
-  "actionSidebar.tooltip.managePermissions.authority": "Whether they have the authority to take actions on their own, or require others with the same multi-sig permission to approve as well.",
-  "actionSidebar.tooltip.editColony.colonyName": "The display name of the Colony.",
-  "actionSidebar.tooltip.editColony.colonyLogo": "The logo of the Colony.",
-  "actionSidebar.tooltip.editColony.colonyDescription": "The short description of the Colony what it is about and it's purpose.",
+  "actionSidebar.tooltip.actionType": "Identifies the type of action.",
+  "actionSidebar.tooltip.decisionMethod": "Mechanism behind the decision-making process.",
+  "actionSidebar.tooltip.createdIn": "Team responsible for decision-making and action execution.",
+  "actionSidebar.tooltip.description": "Rationale and context behind the action.",
+  "actionSidebar.tooltip.singlePayment.from": "Team source of the funds.",
+  "actionSidebar.tooltip.singlePayment.recipient": "Member or address designated to receive the funds.",
+  "actionSidebar.tooltip.singlePayment.amount": "The amount of funds and token type for the payment.",
```

Comment:
As of the spec here https://www.notion.so/colony/Actions-Motions-775cb0e78eb44783a6f28702b7a9dc33?pvs=4#64368b1c05044078a4374f755be26754 and here https://tw.auroracreation.com/app/tasks/15567878?c=5830182

This should be: "Quantity and type of funds of the payment."

Reply:
done